app/components/agenda/agendaitem/agendaitem-decision.js

```diff
@@ -79,7 +79,10 @@ export default class AgendaitemDecisionComponent extends Component {
   });
 
   loadDocuments = task(async () => {
-    let pieces = await this.throttledLoadingService.loadPieces.perform(this.args.agendaitem);
+    let pieces = await this.store.query('piece', {
+      'filter[agendaitems][:id:]': this.args.agendaitem.id,
+      'filter[:has-no:next-piece]': true,
+    });
```

Comment:
**note (non-blocking)**: replacing `throttledLoadingService` should be fine, but was probably there to make use of caching

I think the `throttledLoadingService` was added here https://github.com/kanselarij-vlaanderen/frontend-kaleidos/commit/6e53387d212bea6460f0da1f4c2c483bf6438448 to make use of the caching, since we're already loading the pieces for the agendaitem when opening the agenda. Given that there's extra constraints here (we only want the latest versions) it's probably fine to replace it here.

@ValenberghsSven were there any other reasons to use the `throttledLoadingService` here?